torch/ao/quantization/fx/backend_config_dict/tensorrt.py

```diff
@@ -86,6 +86,19 @@ def get_tensorrt_backend_config_dict():
             weighted_op_qint8_dtype_config,
         ]
     }
+    addmm_config = {
```

Comment:
doesn't have to be in this PR, but is there a README or a type definition of the valid keys of this dictionary, and some validation to catch common errors?  I.e. how would a user know that this new `input_type_to_index` is a valid option and what is its format.

Reply:
yeah planning to have a complete README after this is done. also we might want to use dataclass for these configs as suggested by Dima

Reply:
will add a TODO for now